docs/user-documentation/metadata.md

```diff
@@ -1,83 +1,130 @@
+!!! tip "Nerd note:"
+    The parts of a field are called properties, so 'entity reference' and 'relation type' are properties of the Typed Relation field type.
+
+Islandora Defaults provides a 'Linked Agent' field as part of the Repository Item content type, and populates the available relationships from the MARC relators list.
+![Screenshot of adding a value into a typed relation field](../assets/metadata_typed_relation_field.png)
+
+The list of available relations for this field is configurable at '/admin/structure/types/manage/islandora_object/fields/node.islandora_object.field_linked_agent'. If you re-use this existing field on another content type, you can define different available relations for that instance of the field. Relations are defined in the format `key|value`, and the key is used in the RDF mapping (see below).
 
 ![Screenshot of the 'Available Relations' configuration text box for the 'Linked Agent' field.](../assets/metadata_available_relations_config.png)
 
+!!! tip "Typed Relation fields and mapping to RDF:"
+    Unlike other fields, which can be assigned RDF predicates in RDF Mapping yaml files, a typed relation field uses a different predicate depending on the chosen "type". These predicates are assigned using the 'keys' in the `key|value` configuration. The key must be formated `namespace:predicate`, e.g. `relators:act`. The namespace ('relators') must be pre-defined in code, using a hook. See the RDF Mapping section of [Create / Update a Content Type](content_types.md) for more details, including a list of available pre-defined namespaces.
+
 
 # Getting Metadata into Fedora and a Triple-store
 
-The above sections described how Drupal manages and stores metadata, but the key feature of Islandora 8 is pushing that metadata into a Fedora 4+ repository and a triple-store. Islandora does this by using Drupal's serialization capabilities to provide a JSON-LD serialization that can be ingested by Fedora 4+ repository and triple-stores. In response to write operations, it sends notifications to the repository and triple-store that a node or term is available to ingest.
+The above sections described how Drupal manages and stores metadata, but Islandora 8 provides for pushing that metadata into a Fedora 4+ repository and a triple-store. Islandora does this by using Drupal's serialization capabilities to provide a JSON-LD serialization that can be ingested by Fedora 4+ repository and triple-stores. In response to write operations, it sends notifications to the repository and triple-store that a node or term is available to ingest.
 
 The JSON-LD module works by taking node or term and its corresponding RDF mapping to create a JSON-LD serialization. The RDF mapping for a content type or vocabulary lists its fields and the RDF predicates that should be used for them.
```

Comment:
Not your words but, "taking node or term" is missing an "a".  e.g. "taking a node or term".  Or even better, just say "taking an entity".

Reply:
I know, I rushed that last part. Thanks.

Reply:
it now reads:

The JSON-LD module (written for Islandora) takes an entity and its corresponding RDF mapping (a structure defined by the contributed RDF module) to create a JSON-LD serialization. An RDF mapping for a content type, media type, or vocabulary lists its fields and the RDF predicates that should be used for them.

The JSON-LD serialization for an entity is available by appending `?_format=jsonld` to the entity's URL. This particular syntax takes advantage of the REST API module. Below is an example JSONLD document representing the RDF serialization of a Repository item node created in a standard islandora-playbook based vagrant VM: